website/docusaurus.config.js

```diff
@@ -0,0 +1,68 @@
+module.exports = {
+  title: 'Gatekeeper',
+  tagline: 'Policy Controller for Kubernetes',
+  url: 'https://sozercan.github.io/gatekeeper/website/docs/', // TODO(sertac): update to openpolicyagent before merging
+  baseUrl: '/gatekeeper/website/',
+  onBrokenLinks: 'throw',
+  favicon: 'img/favicon.ico',
+  organizationName: 'sozercan', // TODO(sertac): update to openpolicyagent before merging
```

Comment:
dangling TODO

Reply:
This is known. Will need to update before merge, otherwise preview in my fork will be broken